src/game.cpp

```diff
@@ -6618,7 +6618,17 @@ look_around_result game::look_around( catacurses::window w_info, tripoint &cente
         }
 
         int la_y = 0;
-        int la_x = TERMX - panel_width;
+        int la_x = 0;
+        std::string position = get_option<std::string>( "LOOKAROUND_POSITION" );
+        if( position == "left" ) {
+            if( get_option<std::string>( "SIDEBAR_POSITION" ) == "right" ) {
+                la_x = panel_manager::get_manager().get_width_left();
+            } else {
+                la_x = panel_manager::get_manager().get_width_left() - panel_width;
+            }
+        } else if( position == "right" ) {
```

Comment:
If something goes wrong, it's be better.
```suggestion
        } else {
```

Reply:
It should be fine.
options.cpp only has two selections for the player, "left" and "right", so one of those conditional has to pass. If _somehow_ the player manages to select neither and failing both conditional, la_x is already defined as 0, which will put the look_around window on the left.

Reply:
Yes, but by default it should be on the right side, in my opinion.

Reply:
With that logic, I might as well just remove the whole conditional :man_shrugging: 

So la_x is defined to be on right hand by default, and _if_ the setting of `LOOKAROUND_POSITION` is "left", then la_x will get a new value. Otherwise, it pass through unchanged.